### ISSUE TITLE
VS2022 Run Test  Error

### ISSUE THREAD

**Issue by sg0771** [Unit-X/kcp-cpp#4]
KCP-cpp test
Connecting IP:port > 127.0.0.1:61263
The server is not active.
The server timed out a client.

**Comment by andersc** [comment]
Confirmed. I get the same response using VS2022

```
$ ./kcp_cpp.exe
KCP-cpp test
Connecting IP:port > ::1:54633
The server is not active.
The server timed out a client.
The server is not active.
```

Will try to solve.

/A

**Comment by andersc** [comment]
The rcvfrom method in the client fails with error 10022. The way Windows require a sending UDP socket to be bound to INADDR_ANY and port 0 differs from BSD sockets and Linux sockets. This error for what I can see so far is originating in kissnet. Will work on a solution but it requires some work.

here is more detailed information about the problem ->

https://stackoverflow.com/questions/47383826/winsock2-recvfrom-function-ends-with-error-10022-invalid-argument

**Comment by tomasoh-ateliere** [comment]
This should be solved by PR #5
